File: conditionals/conditional_notes.py
# Malu Estevam, conditional notes

num = 6

if num >  5 and num < 10: # nested condition: not advised to nest past two
    if num == 7:
        print(f"{num} is an unluck number")
    else:
        print(f"{num} is a large single digit number")
else:
    print(f"This is {num}")

File: conditionals/conditional_notes.c
//Malu Estevam, conditional notes C
#include <stdio.h>
#include <string.h>
char name[] = "Malu";
int num[] = 7;

int main(void){
    if(strcmp(name, "Malu") == 0){
        printf("Hello Malulu. Welcome to class\n");
    }else{
        printf("Hello %s, welcome to class!\n", name);
    }
// && = and
// || = or
// != = not

    if(num > 5 && num < 10){
        if(num == 7){
            printf("%d is an unlucky number\n", num);
        }else{
            printf("%d is a large single digit number\n", num);
        }
    }else if (num > 10){
        printf("%d is not a single digit number\n", num);
    }else{
        printf("%d is a small single digit number\n", num);
    }
    return 0;
}